src/inference/include/openvino/runtime/core.hpp

```diff
@@ -547,6 +547,22 @@ class OPENVINO_RUNTIME_API Core {
         return get_property(deviceName, property.name(), {}).template as<T>();
     }
 
+    /**
+     * @brief Gets properties related to device behaviour.
+     *
+     * The method is needed to request the property of specified device from meta plugins.
```

Comment:
The value of the property of specified HW device can be the value set by set_property and cached in meta plugin, if it is not set, the value will be from the HW device.
@ilyachur @wangleis  If this is the message we want to add?